yellow-paper/docs/rollup-circuits/root-rollup.md

```diff
@@ -161,7 +177,7 @@ RootRollupPublicInputs *--Header : header
 
 ```python
 def RootRollupCircuit(
-    l1_to_l2_roots: MessageCompressionBaseOrMergePublicInputs,
+    l1_to_l2_roots: TreeConversionData,
```

Comment:
If this struct isn't just "roots of L1 to L2 message trees", consider renaming it

Reply:
It also got the proof and aggregation object for verification